iroh-bitswap/src/network.rs

```diff
@@ -123,6 +123,10 @@ impl Network {
             for i in 1..=retries {
                 debug!("send:{}: try {}/{}", peer, i, retries);
                 let (s, r) = oneshot::channel();
+                record!(
```

Comment:
this might fail, so we don't actually know if these are the bytes we sent

Reply:
I'm aware, but would rather over then under count here ie we streamed half a response and channel broke.

Reply:
fair